cli.js

```diff
@@ -62,10 +62,27 @@ if (cli.flags.init) {
 
 log.write();
 
-var api = new Api(cli.input, {
+var apiOpts = {
 	failFast: cli.flags.failFast,
 	serial: cli.flags.serial
-});
+};
+
+if (cli.flags.compilers) {
+	var compilers = cli.flags.compilers;
+	if (!Array.isArray(compilers)) {
+		compilers = [compilers];
+	}
+	apiOpts.compilers = {};
+	compilers.forEach(function (compilerDef) {
+		var colonIdx = compilerDef.indexOf(':');
+		if (colonIdx === -1) {
+			throw new Error('compilers must be specified with: --compilers=<ext>:<require-path>');
+		}
+		apiOpts.compilers[compilerDef.substring(0, colonIdx)] = compilerDef.substring(colonIdx + 1);
```

Comment:
I could just use `string.split()` here for a lot less code. But that would break absolute paths on Windows. Mocha just uses `split`. Absolute paths seem unlikely.

Thoughts?